server/snapshot/snapshot.go

```diff
@@ -96,14 +102,11 @@ func contentExpired(gun string, sn *data.SignedSnapshot, store storage.MetaStore
 		if err != nil {
 			return false
 		}
-		roleExp, newHash := roleExpired(curr, meta)
+		roleExp, newMeta := roleExpired(curr, meta)
 		if roleExp {
-			updatedMeta[role] = data.FileMeta{
-				Length: int64(len(curr)),
-				Hashes: data.Hashes{
-					"sha256": newHash,
-				},
-			}
+			updatedMeta[role] = newMeta
+		} else {
+			updatedMeta[role] = meta
```

Comment:
@cyli as we're progressing with the delegations work I'm seeing more and more that this attempt to update the snapshot is a poor idea. I'm just going to remove this code. Getting a "new" snapshot will just resign the previous one with an updated expiry if its expired.